quizapp/src/main/java/billennium/quizapp/entity/Candidate.java

```diff
@@ -3,22 +3,31 @@
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
-import org.hibernate.annotations.Type;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
 
-import javax.persistence.Entity;
-import javax.persistence.FetchType;
-import javax.persistence.Id;
-import javax.persistence.OneToOne;
+import javax.persistence.*;
 import java.util.UUID;
 
 @Entity
-@Data
 @AllArgsConstructor
 @NoArgsConstructor
+@Data
 public class Candidate {
 
     @Id
-    @Type(type = "pg-uuid")
+    @GeneratedValue(
+            strategy = GenerationType.AUTO,
+            generator = "pg-uuid"
+    )
+    @GenericGenerator(
+            name = "pg-uuid",
+            strategy = "uuid2",
+            parameters = @Parameter(
+                    name = "uuid_gen_strategy_class",
+                    value = "com.vladmihalcea.book.hpjp.hibernate.identifier.uuid.PostgreSQLUUIDGenerationStrategy"
+            )
+    )
     private UUID id;
```

Comment:
to już zmieniłem w moim commicie ten GenericGenerator wygląda strasznie :), niestety samo  type=pg-uuid i dodanie Mode do H2 niezałatwia sprawy.

Reply:
Mam dziwne wrażenie, że ten kod jest z PR który jeszcze nie został zmergowany. Czy mam racje ?

Reply:
Dodatkowo w tym innym PR juz tego kodu nie ma